docs/content/deployment/run-retries.mdx

```diff
@@ -5,19 +5,24 @@ description: Automatically retry Dagster runs
 
 # Run Retries
 
-If you configure retries at the [Job](/\_apidocs/jobs#jobs) level, a new run will be kicked off when a run for that job fails. Compared to [Op retries](/concepts/ops-jobs-graphs/op-retries), the max retry limit for run retries applies to the whole run instead of each individual Op. Run retries also handle the cases where a run worker crashed.
+If you configure retries at the run level, a new run will be kicked off whenever a run fails for any reason. Compared to [Op retries](/concepts/ops-jobs-graphs/op-retries), the max retry limit for run retries applies to the whole run instead of each individual Op. Run retries also handle the case where a run worker crashes or is unexpectedly terminated.
 
 ## Configuration
 
-To enable run retries, add the following to your `dagster.yaml`. This will start a new daemon which polls to the event log for run failure events.
+How to configure a default maximum number of retries for all runs in your Dagster deployment depends on whether you're using Dagster Cloud or Dagster Open Source:
+
+- **Dagster Cloud**: Use the [Dagster Cloud UI or the dagster-cloud CLI][cloud-deployment-settings]
+- **Dagster Open Source**: Use your instance's `dagster.yaml`
```

Comment:
@erinkcochran87 this is driving me nuts, do you have any idea why it doesn't like this? I copy-pasted it verbatim from the concurrency docs, but it appears incorrectly and mdx-format auto-corrects it to:
`
**Dagster Cloud**: Use the \[Dagster Cloud UI or the dagster-cloud CLI]\[cloud-deployment-settings]